### PR TITLE
Adding Functionality of checking is string contains only ascii characters

### PR DESCRIPTION
**Pull request by JavidSumra**
By Making This PR I am Solving Issue #14 
I am Contributing Under JWOC 2024

### CODE REVIEW

**Comment by jagroop-dev-11** [comment]
Please remove linting and also add these methods in readme.md file in the same PR  @JavidSumra

**Comment by JavidSumra** [comment]
Hey @Jagroop2001 why do we need to remove the linting

**Comment by JavidSumra** [comment]
Hey @Jagroop2001 why do we need to remove the linting?

**Comment by jagroop-dev-11** [comment]
So that only that changes are merged that you have changed. Not other's otherwise it cause issues of merge conflict to others 
@JavidSumra

**Comment by JavidSumra** [comment]
Hey @Jagroop2001 Sir Check now?

File: src/utils/utilityFunctions.ts [hunk 1]
```diff
@@ -1,95 +1,97 @@
...
 export const isIdentical = (strOne: string, strTwo: string) => {
-    strOne = strOne.trim();
-    strTwo = strTwo.trim();
-    if (strOne.toLowerCase() === strTwo.toLowerCase()) {
-        return true;
-    }
-    return false;
+  strOne = strOne.trim();
+  strTwo = strTwo.trim();
+  if (strOne.toLowerCase() === strTwo.toLowerCase()) {
```
Comment (Jagroop2001) [inline src/utils/utilityFunctions.ts:5]: Still available @JavidSumra

**Comment by JavidSumra** [comment]
@Jagroop2001 Sir Check Now?

**Comment by jagroop-dev-11** [comment]
I mean remove all the linting in  src/utils/utilityFunctions.ts page.
Just do one thing, copy code from master and add your's.
Only your code changes must be visible .

I hope it's clear to your 😊

**Comment by JavidSumra** [comment]
Hello, @Jagroop2001, Sir, I've accomplished the same task.

**Comment by JavidSumra** [comment]
Hey @Jagroop2001 Sir Please Have a Look Now..

**Pull request merged**